app/views/admin/configs/show.html.erb

```diff
@@ -200,7 +200,7 @@
                         <div
                           class="enabled-indicator <%= SiteConfig.allow_email_password_registration ? "visible" : "" %>"
                           data-config-target="enabledIndicator">
-                          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon admin-config-checkmark", title: "Checkmark") %>
+                          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon admin-config-checkmark", title: "Email enabled") %>
```

Comment:
The change is for accessibility, but it also simplifies the test by getting by title via Cypress testing library APIs.